config/logcollector_manifest_normal

```diff
@@ -0,0 +1,33 @@
+echo,### Probing Directories ###
```

Comment:
This file is identical to IID's [manifest file](https://github.com/Azure/azure-diskinspect-service/blob/master/pyServer/manifests/linux/agents), collection option agents.

Reply:
Why not keep it simple and just collect full everytime? We already have the MUST_HAVE files given priority

Reply:
The periodic collection should not be full due to resource usage on the VM, but we want to provide the full option for on-demand use of the tool as a command line utility for support engineers to use to pull a full set of logs from the VM. These two options are already given by Windows Agent and its log collection tool.